woocommerce/woocommerce
# Add icon support to product section block

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/37264



### How to test the changes in this Pull Request:




1. Got to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
2. Under `Features` tab make sure to enable `block-editor-feature-enabled`
3. Then visit `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. An icon should be shown right before the `Basic details` section title.

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/13334210/226940806-dca17f86-3ac4-4dff-9cb0-9d7b5f10b385.png"> 



### Other information:




### FOR PR REVIEWER ONLY:
